jet_tracking/jt_utils.py

```diff
@@ -103,23 +112,26 @@ def fit_cspad(azav, norm, gas_det):
 
 
 def get_cspad(azav, r, gas_det):
-    '''
-    Get the intensity of the diffraction ring on the CSPAD
+    """
+    Get the intensity of the diffraction ring on the CSPAD.
 
     Parameters
     ----------
     azav : ndarray
-        Azimuthal average calculated from CSPAD
+        Azimuthal average calculated from CSPAD.
+
     r : int
-        radius of diffraction ring
+        Radius of diffraction ring.
+
     gas_det : float
-        gas detector
+        Gas detector intensity.
 
     Returns
     -------
     intensity : float
-        sum of qbins 5 above and below the center, normalized by gas detector
-    '''
+        Sum of 5 qbins above and below the center, normalized by gas detector.
+    """
+
     intensity = sum(azav[r - 5:r + 5]) / gas_det
```

Comment:
The bins around the peak radius will be passed in from the calibration data.  Make that a param of this function.

Reply:
Take a look at af9029a. Is that kinda what you were thinking?